manager/src/static/js/marks.js

```diff
@@ -34,11 +33,11 @@ function updateTooltipContent(mark, o, show_telemetry) {
             key: `${key}.${nestedKey}`,
             value: nestedValue,
           }))
-        : { key, value },
+        : { key, value }
   );
 
   persistentDataArray.forEach(({ key, value }) =>
-    addOrUpdateTableRow(table, key, value),
+    addOrUpdateTableRow(table, key, value)
```

Comment:
Please keep the coding convention used in js codebase so far.

The rationale behind:
- Consistency: JS codebase is already using this convention, let's keep it consistent. Also, many style guides and tools (Prettier, ESLint) recommend trailing commas for multi-line objects and arrays.
- Cleaner diffs:
When adding or removing items, only the changed line is modified in version control, making diffs easier to read.
- Easier editing:
You can add new elements without worrying about whether to add or remove a comma from the previous line.


```suggestion
    addOrUpdateTableRow(table, key, value),
```